### PR TITLE
Drop support for EOL Python 3.8

### PR DESCRIPTION
**Pull request by hugovk**
Fixes https://github.com/indygreg/python-build-standalone/issues/342.

This will help reduce the huge build matrix, which will need to grow fairly soon when 3.14 is added.

### CODE REVIEW

File: cpython-windows/build.py [hunk 1]
```diff
@@ -614,230 +614,6 @@ def hack_project_files(
         pass
 
 
-PYPORT_EXPORT_SEARCH_39 = b"""
```
Comment (hugovk) [inline cpython-windows/build.py:617]: The use of these constants were removed in https://github.com/indygreg/python-build-standalone/issues/342.

**Comment by zanieb** [comment]
Note I already removed this from the build matrix in #344

**Comment by zanieb** [comment]
@hugovk would you update with `main` for the Windows fixes?

**Review by zanieb — APPROVED** [review]
Awesome

**Pull request merged**